TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/netty/FrontsideHandler.java

```diff
@@ -0,0 +1,110 @@
+package org.opensearch.migrations.trafficcapture.proxyserver.netty;
+
+import io.netty.bootstrap.Bootstrap;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelFutureListener;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.channel.ChannelOption;
+import io.netty.handler.logging.LogLevel;
+import io.netty.handler.logging.LoggingHandler;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class FrontsideHandler extends ChannelInboundHandlerAdapter {
+
+    private Channel outboundChannel;
+
+    private final String host;
+    private final int port;
+
+    /**
+     * Create a handler that sets the autoreleases flag
+     * @param host
+     * @param port
+     */
+    public FrontsideHandler(String host, int port) {
+        this.host = host;
+        this.port = port;
+    }
+
+    @Override
+    public void channelActive(ChannelHandlerContext ctx) {
+        final Channel inboundChannel = ctx.channel();
+        // Start the connection attempt.
+        Bootstrap b = new Bootstrap();
+        b.group(inboundChannel.eventLoop())
+                .channel(ctx.channel().getClass())
+                .handler(new LoggingHandler(LogLevel.ERROR))
```

Comment:
Netty novice question: this isn't a handler that's going to cause everything that comes through the channel to be logged, right?

Reply:
It DOES log everything on the channel to the log4j facilities - meaning with the current setup, stderr.  In a future commit, I've removed the handler - or commented it out entirely.